rain_api_core/auth.py

```diff
@@ -0,0 +1,154 @@
+import contextlib
+import dataclasses
+import json
+import urllib.request
+from http.cookies import CookieError, SimpleCookie
+from time import time
+from typing import List, Mapping, Union
+from wsgiref.handlers import format_date_time as format_7231_date
+
+import jwt
+
+from rain_api_core.logging import get_log
+
+log = get_log()
```

Comment:
I don't think you want to be using `get_log`. This is more for creating a root logger on the application level. You want to just use normal `logging.getLogger(__name__)` here. That's how other files in rain-api-core do it.